client/gutenberg/extensions/visited/editor.scss

```diff
@@ -0,0 +1,8 @@
+.wp-block-jetpack-visited-inner-block .components-notice {
+  margin: 0;
+
+  .components-notice__content {
+    margin: .5em 0;
+    font-size: .8em;
+  }
+}
```

Comment:
I don't think we should be overriding these styles in a per block basis. We should fix this in Gutenberg.